website/static/js/index.js

```diff
@@ -31,4 +31,4 @@ window.startup = function() {
     });
 
     doDemo();
-};
+})();
```

Comment:
I take it the `<script defer`s are guaranteed to execute in order, and only after the DOM has loaded?  [Yep](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script).  Cool.